config/analytics.php

```diff
@@ -46,6 +46,15 @@
         // '/users/*',
     ],
 
+    /**
+     * Ignore methods.
+     *
+     * The HTTP verbs/methods that should be excluded from page view tracking.
+     */
+    'ignoreMethods' => [
```

Comment:
Can you change this to use `snake_case`?

Reply:
I also prefer snake case, but opted for camel case due to the existence of `ignoredIPs` and `ignoreRobots`.

Unless there's a good reason to break the current convention.

Reply:
That's probably my fault, sorry. I also prefer snake case to camel case, but would rather have consistency here and not introduce any breaking changes. Thanks both 👍